src/components/base/JRaisedButton/JRaisedButton.js

```diff
@@ -1,14 +1,14 @@
 // @flow
 
-import React from 'react'
+import React, { PureComponent } from 'react'
 import classNames from 'classnames'
+import handle from 'utils/eventHandlers/handle'
```

Comment:
@flexxnn @Romanior @Dragonesis
Hey guys, let use only `git rebase` instead of `git merge`
Also, if you do `merge`, please check result, to avoid such mistakes as this one:
lines 5 & 7 are equals

Reply:
this likely happens b/c @Dragonesis Rebased from @IViol branch, we should not use rebase.

Reply:
Duplicated line has been removed